schema/v1/trigger-event.json

```diff
@@ -0,0 +1,188 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Trigger Event",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "type": "string"
+    },
+    "id": {
+      "type": "string",
+      "description": "The unique identifier of the trigger event."
+    },
+    "eventClassName": {
+      "type": "string",
+      "description": "A fully qualified class name for the trigger event."
+    },
+    "eventBusinessName": {
```

Comment:
@eacarvalho what is this used for? I seems that EO does not use it.

Reply:
Used by Engage App: Business name of the event, displayed in Engage App